api/src/templates/new_low_income_member.body.html

```diff
@@ -0,0 +1,10 @@
+<h2>
+  #{{member.member_number}} {{member.firstname}} {{member.lastname}} has
```

Comment:
Might be useful with a link directly to the member

```suggestion
   <a href="https://admin.makerspace.se/membership/members/{{member.member_id}}">#{{member.member_number}} {{member.firstname}} {{member.lastname}}</a> has
```